Internet Store/Internet Store/Services/AssortimentService.cs

```diff
@@ -0,0 +1,66 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Internet_Store.Models;
+
+namespace Internet_Store.Services
+{
+    /// <summary>
+    /// The singleton pattern logic in a class of shopping busket.
+    /// </summary>
+    internal sealed class AssortimentService
```

Comment:
AssortimentService is also Singleton? Cant see why can we just use different services, since the logic is indifferent for any of the incoming parameters